.github/workfows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  run:
+    name: build
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          #- windows-latest
+          #- macos-latest
+        ocaml-compiler:
+          - 4.03.x
+          - 4.13.x
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: ${{ matrix.ocaml-compiler }}
+    - run: opam pin -n .
+    - run: opam depext -yt murmur3
```

Comment:
@rr0gi I changed the name of the package to `murmur3` from `ocaml-murmur3` - would you rather keep the name?